### PR TITLE
rename `hc` to `satellite` for continuity

### PR DESCRIPTION
**Pull request by dylanlott**
renames hc to satellite for reasons

---
This change is [Reviewable](https://reviewable.io/reviews/storj/storj/281)

### CODE REVIEW

File: cmd/satellite/main.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ import (
 
 var (
 	rootCmd = &cobra.Command{
-		Use:   "hc",
+		Use:   "satellite",
 		Short: "Heavy client",
```
Comment (kaloyan-raev) [inline cmd/satellite/main.go]: Should we change this too?

File: cmd/satellite/Dockerfile [hunk 2]
```diff
@@ -19,7 +19,7 @@ ENV REDIS_ADDR=redis:6379 \
     HTTP_PORT=:8080 \
     BOOTSTRAP_ADDR=bootstrap.storj.io:8080
 WORKDIR /app
-COPY --from=build-env /go/src/storj.io/storj/cmd/hc/satellite /app/
+COPY --from=build-env /go/src/storj.io/storj/cmd/satellite/satellite /app/
 COPY cmd/hc/entrypoint /entrypoint
```
Comment (kaloyan-raev) [inline cmd/satellite/Dockerfile]: Is this `cmd/hc/entrypoint` still good to remain like this?

**Comment by iglesiasbrandon** [comment]
closing this to merge Kishore's changes instead

**Comment by bryanchriswhite** [comment]
^ #282

**Pull request closed without merging**